arc/conformance/plugin/arc_conformance.sh

```diff
@@ -155,6 +161,32 @@ setupKeyVault() {
   --file testec.pfx 2> "${results_dir}"/error || python3 /arc/setup_failure_handler.py
 }
 
+# setup kubeconfig for conformance test
+setupKubeConfig() {
+  APISERVER=https://kubernetes.default.svc/
+  TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+  cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt > ca.crt
+
+  kubectl config set-cluster azure-arc-akv-test \
+    --embed-certs=true \
+    --server=${APISERVER} \
+    --certificate-authority=./ca.crt 2> "${results_dir}"/error || python3 /arc/setup_failure_handler.py
+
+  kubectl config set-credentials azure-arc-akv-test --token="${TOKEN}" 2> "${results_dir}"/error || python3 /arc/setup_failure_handler.py
+
+  # Delete previous rolebinding if exists. And ignore the error if not found.
+  kubectl delete clusterrolebinding clusterconnect-binding || true
+  kubectl create clusterrolebinding clusterconnect-binding --clusterrole=cluster-admin --user="${OBJECT_ID}" 2> "${results_dir}"/error || python3 /arc/setup_failure_handler.py
+
+  kubectl config set-context azure-arc-akv-test \
```

Comment:
`azure-arc-akv-test` is used in a number of places. Maybe make it a constant and reuse it?